### PR TITLE
Migrate PSK to Polymer 2.0

### PR DESCRIPTION
**Pull request by abdonrd**
This is a work in progress pull request. :)

### CODE REVIEW

**Comment by FluorescentHallucinogen** [comment]
@abdonrd Have you tried to build it?

I've tried to build current [`2.0-preview`](https://github.com/PolymerElements/polymer-starter-kit/tree/0c12fc9901142afae027ea5e554af700f92b1d63) and [`master`](https://github.com/PolymerElements/polymer-starter-kit/tree/369af2b968c77a730c453aae5f9dc3b580e874c3) branches, both do not work.

I believe the problem is in https://github.com/PolymerElements/polymer-starter-kit/pull/1008.

**Comment by abdonrd** [comment]
@FluorescentHallucinogen yes, it doesn't work right now. It is work in progress.

References to https://github.com/PolymerElements/polymer-starter-kit/pull/1008 and https://github.com/PolymerElements/polymer-starter-kit/pull/1019 about the relative paths.

File: src/my-app.html [hunk 1]
```diff
@@ -101,49 +111,78 @@
...
+        };
+      }
+
+      static get observers() {
+        return [
+          '_routePageChanged(routeData.page)',
+        ];
+      }
+
+      computeRootPattern(rootPath) {
```
Comment (danizord) [inline src/my-app.html]: Should be prefixed with `_`?

File: src/my-app.html [hunk 2]
```diff
@@ -22,6 +22,11 @@
 <link rel="import" href="../bower_components/paper-icon-button/paper-icon-button.html">
 <link rel="import" href="my-icons.html">
 
+<link rel="lazy-import" href="my-view1.html">
+<link rel="lazy-import" href="my-view2.html">
+<link rel="lazy-import" href="my-view3.html">
+<link rel="lazy-import" href="my-view404.html">
```
Comment (danizord) [inline src/my-app.html]: Should not it be inside template?

Comment (abdonrd) [inline src/my-app.html]: Nop! The [lazy-imports](https://github.com/Polymer/lazy-imports) has the `lazy-import` outside the `template`.

File: src/my-app.html (hunk 1 above)
Comment (abdonrd) [inline src/my-app.html]: Probably. But first we are waiting for https://github.com/PolymerElements/polymer-starter-kit/pull/1019#issuecomment-306350626.

**Comment by abdonrd** [comment]
We probably merge after #998.

**Pull request merged**